### PR TITLE
factories.py

### PR DESCRIPTION
**Pull request by yevhen-m**
factories.py (factory boy) running is embedded into `fab init_db`

Context manager for sqlalchemy sessions is no longer used (it's now clear to me that we don't need to use it).

### CODE REVIEW

File: ecomap/api/v1_0/models.py [hunk 1]
```diff
@@ -1,169 +1,205 @@
...
+           nullable=False)
+)
+
 
 class ProblemType(Base):
     __tablename__ = 'problem_types'
 
     id = Column(Integer, primary_key=True)
-    type = Column(String(100), nullable=False)
+    type = Column(String, nullable=False)
```
Comment (ITsvetkoFF) [inline ecomap/api/v1_0/models.py:40]: please leave  all string sizes

File: ecomap/api/v1_0/models.py [hunk 2]
```diff
@@ -1,169 +1,205 @@
...
 
 
 class Role(Base):
     __tablename__ = 'roles'
 
     id = Column(Integer, primary_key=True)
```
Comment (ITsvetkoFF) [inline ecomap/api/v1_0/models.py:46]: we need autoincrement="true" here and anywhere maybe)

File: ecomap/api/v1_0/models.py [hunk 3]
```diff
@@ -1,169 +1,205 @@
...
-    role_id = Column(Integer, ForeignKey(u'roles.id', ondelete=u'CASCADE'), primary_key=True, nullable=False)
+    roles = relationship('Role', secondary=user_roles)
 
 
 class Problem(Base):
     __tablename__ = 'problems'
 
     id = Column(Integer, primary_key=True)
-    title = Column(String(130), nullable=False)
+    title = Column(String)
```
Comment (ITsvetkoFF) [inline ecomap/api/v1_0/models.py:79]: nullable=False это справедливо

File: ecomap/api/v1_0/models.py (hunk 2 above)
Comment (yevhen-m) [inline ecomap/api/v1_0/models.py:46]: are you sure? it works fine for me.
just check and if it doesn't work for you, i'll add autoincrements.

File: ecomap/api/v1_0/models.py (hunk 3 above)
Comment (yevhen-m) [inline ecomap/api/v1_0/models.py:79]: agree

**Pull request closed without merging**